packages/dm-core-plugins/src/job/RemoveJobDialog.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { Button } from '@equinor/eds-core-react'
+import { Dialog } from '@development-framework/dm-core'
+
+type RemoveJobDialogProps = {
+  isOpen: boolean
+  remove: () => void
+  afterRemove: () => void
+}
+export const RemoveJobDialog = (props: RemoveJobDialogProps) => {
+  const { isOpen, remove, afterRemove } = props
+  return (
+    <Dialog open={isOpen}>
+      <Dialog.CustomContent>
+        <pre> Are you sure you want to stop the current job? </pre>
```

Comment:
\<pre\> is not the preferred html element for standard text...
Please change to "Typography" or \<p\>

Reply:
<img width="371" alt="image" src="https://github.com/equinor/dm-core-packages/assets/43639886/70270822-2f52-4f1b-9f27-571fa905503f">

Reply:
<img width="407" alt="image" src="https://github.com/equinor/dm-core-packages/assets/43639886/e284df63-1dfc-4f6f-96b3-6db48d331e99">